munge_sumstats.py

```diff
@@ -440,111 +443,129 @@ def allele_merge(dat, alleles, log):
                     help='Name of signed sumstat column, comma null value (e.g., Z,0 or OR,1). NB: case insensitive.')
-parser.add_argument('--info', default=None, type=str,
+iformat.add_argument('--info', default=None, type=str,
                     help='Name of INFO column (if not a name that ldsc understands). NB: case insensitive.')
-parser.add_argument('--info-list', default=None, type=str,
+iformat.add_argument('--info-list', default=None, type=str,
                     help='Comma-separated list of INFO columns. Will filter on the mean. NB: case insensitive.')
-parser.add_argument('--nstudy', default=None, type=str,
+iformat.add_argument('--nstudy', default=None, type=str,
                     help='Name of NSTUDY column (if not a name that ldsc understands). NB: case insensitive.')
-parser.add_argument('--nstudy-min', default=None, type=float,
+iformat.add_argument('--nstudy-min', default=None, type=float,
                     help='Minimum # of studies. Default is to remove everything below the max, unless there is an N column,'
                     ' in which case do nothing.')
-parser.add_argument('--ignore', default=None, type=str,
+iformat.add_argument('--ignore', default=None, type=str,
                     help='Comma-separated list of column names to ignore.')
-parser.add_argument('--a1-inc', default=False, action='store_true',
+iformat.add_argument('--a1-inc', default=False, action='store_true',
                     help='A1 is the increasing allele.')
-parser.add_argument('--keep-maf', default=False, action='store_true',
+
+## filters
+filters = parser.add_argument_group(title="Data Filters", description="Options to apply data filters to summary statistics.")
+filters.add_argument('--maf-min', default=0.01, type=float, help="Minimum MAF.")
+filters.add_argument('--info-min', default=0.9, type=float, help="Minimum INFO score.")
+filters.add_argument('--daner', default=False, action='store_true',
+                    help="Use this flag to parse Stephan Ripke's daner* file format.")
+filters.add_argument('--daner-n', default=False, action='store_true',
+                    help="Use this flag to parse more recent daner* formatted files, which "
+            "include sample size column 'Nca' and 'Nco'.")
+filters.add_argument('--merge-alleles', default=None, type=str,
+                    help="Same as --merge, except the file should have three columns: SNP, A1, A2, "
+                    "and all alleles will be matched to the --merge-alleles file alleles.")
+filters.add_argument('--n-min', default=None, type=float,
+                    help='Minimum N (sample size). Default is (90th percentile N) / 1.5')
+filters.add_argument('--chunksize', default=5e6, type=int,
+                    help='Chunksize.')
+parser.add_argument('--keep-str-ambig', default=False, action='store_true',
+                    help=argparse.SUPPRESS) # This options allows munge sumstats to retain strand ambiguous SNPS instead of dropping them.
```

Comment:
why not put this message in `help`?

Reply:
Hi @tpoterba,

We added this flag so that it doesn't remove strand ambiguous alleles (TA, GC) by default. However, Raymond suggested that for the purposes of generic munging, this option would be better if kept invisible.